docs/README.rst

```diff
@@ -0,0 +1,163 @@
+.. _readme:
+
+pycharm-formula
+============
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/pycharm-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/pycharm-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+Formula to install Pycharm on GNU/Linux and MacOS.
+
+.. contents:: **Table of Contents**
+
```

Comment:
Using `depth` after your PR to the `template-formula`(!), will be resolved when commit is added here:

https://github.com/myii/jetbrains-pycharm-formula/commit/a3bac46edc0678d4436b425e3929d5c04aea72f6#diff-8ab5636c6c18641e5c58358ef489e0f7R20